### ISSUE TITLE
SHREC 2017_track1 setting problems

### ISSUE THREAD

**Issue by xsstef** [kanezaki/SHREC2017_track3#2]
Hello! Thank you for sharing your method, and I have some questions about SHREC 2017 track 1.

In the track report of track1, you mentioned that RotationNet uses 3308 CAD models in ShapeNetSem for training. Then you fine-tune the model using 830 objects in SceneNN. 

I wander to know how to render objects in SNN(RGB-D objects in *.ply format) into 20 views. 
And would you mind teling me your network parameters setting in track1, like batchsize, learning rate..?

Looking forward to your reply!
